### ISSUE TITLE
Current usage of TreeBuilder::root method is deprecated

### ISSUE THREAD

**Issue by javiereguiluz** [liip/LiipFunctionalTestBundle#513]
Sorry if this was reported earlier, but I can't find any issue or PR.

### Steps to reproduce
1. Upgrade to Symfony 4.3 and you'll see this deprecation message:

```
The "Symfony\Component\Config\Definition\Builder\TreeBuilder::root()"
method called for the "liip_functional_test" configuration is deprecated 
since Symfony 4.3, pass the root name to the constructor instead.
```

This is something all Symfony bundles must fix. Here are some examples of how to do it:

* https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/594/files
* https://github.com/doctrine/DoctrineCacheBundle/pull/148/files

**Comment by alexislefebvre** [comment]
It should have been fixed in #478, I'll see why there's still a deprecation.

**Comment by javiereguiluz** [comment]
Closing as fixe by the linked PR. Thanks.
